### ISSUE TITLE
tekton-triggers-controller/core-intereptors/webhook v0.21.0 is forbidden: violates PodSecurity

### ISSUE THREAD

**Issue by CJCShadowsan** [tektoncd/triggers#1479]
# Expected Behavior
Should deploy without errors

# Actual Behavior
deployment comes back with:

```
Error creating: pods "tekton-triggers-controller-99b9d97fc-ps6n2" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (container "tekton-triggers-controller" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "tekton-triggers-controller" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "tekton-triggers-controller" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")

Error creating: pods "tekton-triggers-core-interceptors-856fdf766c-9s8mh" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (container "tekton-triggers-core-interceptors" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "tekton-triggers-core-interceptors" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "tekton-triggers-core-interceptors" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")

Error creating: pods "tekton-triggers-webhook-7bb8db96cc-pdgbk" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (container "webhook" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "webhook" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "webhook" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
```

# Steps to Reproduce the Problem

1. Deploy pipelines v0.41.0, triggers v0.21.0, interceptors v0.21.0, and dashboard v0.30.0

# Additional Info

- Kubernetes version:

```
Server Version: v1.23.12+k3s1
```

- Tekton Pipeline version:

v0.41.0

**Comment by AlanGreene** [comment]
This looks like a duplicate of https://github.com/tektoncd/triggers/issues/1447 which is fixed on main and would be included in the next release.

**Comment by dibyom** [comment]
This should be fixed by #1447
